student_class.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'unknown', parent_permission: true)
+    super(name, age, parent_permission)
```

Comment:
- [ ] Since we are using the colon notation for parent_permission, we need to also change they way it is being used in the **super** method.

- [ ] Also, kindly re-arrange your parameters here.

  ```
  super(age, name, parent_permission: parent_permission)
  ```